client-mr/core/src/main/java/org/apache/hadoop/mapreduce/RssMRUtils.java

```diff
@@ -83,37 +83,37 @@ public static TaskAttemptID createMRTaskAttemptId(
   public static ShuffleWriteClient createShuffleClient(JobConf jobConf) {
     int heartBeatThreadNum =
```

Comment:
I think we could create a rssConf based on the jobConf, and then use the RssConf's function like get() to retrieve the conf value.